addons/hr_holidays/models/hr_leave.py

```diff
@@ -346,11 +345,6 @@ def _search_description(self, operator, value):
         query = self.sudo()._search(domain)
         return [('id', 'inselect', query.select())]
 
-    @api.depends('holiday_status_id')
-    def _compute_state(self):
```

Comment:
Maybe it doesn't crash because the field is already initialised in default_get, so the compute method is not actually triggered? But in any case indeed the compute should be removed and just replaced with a default value